app/models/jurisdiction_membership.rb

```diff
@@ -3,9 +3,29 @@ class JurisdictionMembership < ApplicationRecord
   belongs_to :user
 
   after_commit :reindex_jurisdiction, on: %i[create update destroy]
+  after_commit :create_jurisdiction_collaborator, on: %i[create update]
+  after_commit :destroy_jurisdiction_collaborator, on: %i[update]
```

Comment:
both create and destroy jurisdiction collaborator on update? Is this right?

Reply:
@bkmaibach good catch. Mistake from copying over previous line